lib/benchmark/ips/job.rb

```diff
@@ -88,6 +92,28 @@ def config opts
         @iterations = opts[:iterations] if opts[:iterations]
         @stats = opts[:stats] if opts[:stats]
         @confidence = opts[:confidence] if opts[:confidence]
+        self.quiet = opts[:quiet]
+        self.suite = opts[:suite]
+      end
+
+      def quiet=(val)
+        set_reporter(quiet: val)
+      end
+
+      def suite=(suite)
+        if suite
+          @suite = suite
+        else
+          @suite = Benchmark::IPS::NoopSuite.new
+        end
```

Comment:
is it possible to simplify this?
```suggestion
        @suite = suite || Benchmark::IPS::NoopSuite.new
```

Reply:
I like this - thanks!